frontend/src/main/components/Users/UsersTable.js

```diff
@@ -1,8 +1,17 @@
-
 import OurTable, { ButtonColumn } from "main/components/OurTable"
 import { useBackendMutation } from "main/utils/useBackend";
+import React, { useState } from 'react'; // Added missing imports
+
+
 
 export default function UsersTable({ users}) {
+    // const [sortedUsers, setSortedUsers] = useState(users);
```

Comment:
Commented out code?  https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html#why-should-commented-out-code-be-removed